libs/openchallenges/home/src/lib/home.component.html

```diff
@@ -1,4 +1,21 @@
 <main class="base mat-typography">
+  <div id="announcement">
+    <h4>
+      Welcome to
+      <a href="/about" target="_blank">OpenChallenges</a>!
+    </h4>
+    <h3>This website uses a preview snapshot of the OpenChallenges (OC) Database</h3>
+    <h4>
+      We are working hard to update the OC Database with more data in the upcoming months. A more
+      comprehensive database will be released in September 2023. We encourage organizations to
+      <a
+        href="https://docs.google.com/forms/d/e/1FAIpQLSdLZ6C5ZIzLZS147bmvNm2eJ3kq_88ecHS7ADjD4KaapGhO8g/viewform"
+        target="_blank"
+        >submit new challenges</a
+      >
+      and update existing ones to ensure that their challenge data are comprehensive.
```

Comment:
```suggestion
      and update existing ones to ensure that their profiles are complete.
```

I personally don't like when certain words are used over and over ("comprehensive" in this case) 😅 .  What about this instead?